tools/testing/test_selections.py

```diff
@@ -58,37 +58,43 @@ def _calculate_job_times(reports: List["Report"]) -> Dict[str, float]:
 
 
 def calculate_shards(
-    num_shards: int, tests: List[str], job_times: Dict[str, float]
+    num_shards: int, test_files: List[str], test_file_times: Dict[str, float]
```

Comment:
I didn't change any logic in this function; just renamed the variables to more closely match what we're doing.